### ISSUE TITLE
ReactXP not compatible with react-native 0.65 without patch

### ISSUE THREAD

**Issue by mikehardy** [microsoft/reactxp#1319]
AccessibilityInfo.fetch was finally removed, this PR needs a merge+release for ReactXP to work with react-native 0.65+: #1126 

If anyone else runs into this, there's a patch-package / https://github.com/ds300/patch-package format patch attached on the PR, works

**Comment by mikehardy** [comment]
@fbartho @deregtd @berickson1 @msfterictraut sincere apologies for the direct ping just wanted to mention that without the pull request linked here, which was reviewed and only on pause while we waited for a compatibility window (that has passed I think...) reactxp does not work at all with react-native 0.65+ as the deprecated accessibility property the PR moves away from was finally dropped

**Comment by berickson1** [comment]
To be perfectly honest, I haven't kept up with react-native for the last year or so due to shifting job responsibilities. That being said, it does sound reasonable to make this breaking change that bumps the minimum RN version and change the usage of this API

**Comment by mikehardy** [comment]
Thanks @berickson1 - I just saw @erictraut merge some dependabot PRs using that handle vs mserictraut, perhaps he's the best ping with that handle :-)

**Comment by kg-currenxie** [comment]
Same here. Needing patch to upgrade to 0.65

**Comment by realityworks** [comment]
Project is no longer supported... This is why cross platform is dangerous.

**Comment by mikehardy** [comment]
> This is why cross platform is dangerous.

I love making sweeping generalizations from one project as well! Open source comments are also super dangerous ;-)

I think in reality it is just that react-native-web "won" the mindshare here, at the same time that microsoft poured a bunch of resources into real macos and windows ports (have you seen those? they're quite lively!) meaning this repo is just not really needed.

Best part though: it just enabled you to write pretty-much standard react-native code which is still dangerously cross platform, so you can probably just drop your code right into a react-native / react-native-web template (try https://github.com/plaut-ro/luna but be careful, it's cross-platform) and it'll go.

**Comment by mikehardy** [comment]
Here's some more dangerously cross-platform stuff, one line command to init react-native-web + typescript + firebase with a fully built-out authentication signed-out/signed-in flow https://github.com/invertase/react-native-firebase-authentication-example/ - if you're not careful it *will* work on all platforms though, so take care using it
